test/IDE/complete_override.swift

```diff
@@ -115,9 +115,6 @@ protocol ProtocolA {
   func protoAFunc()
   @objc optional func protoAFuncOptional()
 
-  @noreturn
-  func protoAFuncWithAttr()
-
```

Comment:
Also: you should change CodeCompletion.cpp.  There are two places that set:
```
Options.ExclusiveAttrList.push_back(DAK_NoReturn);
```

After this change it should be
```
Options.SkipAttributes = true;
```